packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -167,7 +167,10 @@ public static void setCustomClientBuilder(CustomClientBuilder ccb) {
     customClientBuilder = ccb;
   }
 
-  public static interface CustomClientBuilder {
+  /**
+   * @deprecated To be removed in a future release. See https://github.com/facebook/react-native/pull/37798#pullrequestreview-1518338914
+   */
+  public static interface CustomClientBuilder extends com.facebook.react.modules.network.CustomClientBuilder {
```

Comment:
Apologies, I'm not too sure what you mean by this. Does this mean you'd like `com.facebook.react.modules.network.CustomClientBuilder` explicitly imported at the top of this file?

Reply:
Anywhere in this Java file where it references `CustomClientBuilder` it will use the old deprecated interface, you could point at the shared interface instead, and will prevent people from upgrading their code.

Reply:
Hi @javache, I believe I've implemented what you'd like. I've changed:

`public static void setCustomClientBuilder(CustomClientBuilder ccb) {`
to
`public static void setCustomClientBuilder(com.facebook.react.modules.network.CustomClientBuilder ccb) {`

Does this cover what you've requested? Anything else, just let me know 🙂